modules/installation-osp-api-scaling.adoc

```diff
@@ -0,0 +1,82 @@
+// Module included in the following assemblies:
+//
+// * networking/openstack/load-balancing-openstack.adoc
+
+[id="installation-osp-api-scaling_{context}"]
+= Scaling clusters by using Octavia
+
+If you want to use multiple API load balancers, or if your cluster does not use Kuryr, create an Octavia load balancer and then configure your cluster to use it.
+
+.Prerequisites
+
+* Octavia is available on your {rh-openstack} deployment.
+
+.Procedure
+
+. From a command line, create an Octavia load balancer that uses the Amphora driver: 
++
+[source,terminal]
+----
+$ openstack loadbalancer create --name API_OCP_CLUSTER --vip-subnet-id <id_of_worker_vms_subnet>
+----
++
+You can use a name of your choice instead of `API_OCP_CLUSTER`.
+
+. After the load balancer becomes active, create listeners:
++
+[source,terminal]
+----
+$ openstack loadbalancer listener create --name API_OCP_CLUSTER_6443 --protocol HTTPS--protocol-port 6443 API_OCP_CLUSTER
+----
++
+[NOTE]
+====
+To view the load balancer's status, enter `openstack loadbalancer list`.
```

Comment:
To avoid the apostrophe (ISG), maybe:

To view the status of the load balancer, enter `openstack loadbalancer list`.